ui/app/mainui/AppMain.qml

```diff
@@ -250,20 +343,14 @@ Item {
         }
 
         function onOpenLink(link: string) {
-            console.warn("opening external url without asking user")
-
             // Qt sometimes inserts random HTML tags; and this will break on invalid URL inside QDesktopServices::openUrl(link)
             link = appMain.rootStore.plainText(link)
 
-            if (appMain.rootStore.showBrowserSelector) {
-                popups.openChooseBrowserPopup(link)
+            const domain = StringUtils.extractDomainFromLink(link)
```

Comment:
hmmm. Probably your idea is better, but I'll share my thought anyway.
For link previews we actually have a `hostname` property, maybe we should reuse it here?

Reply:
Yeah, if it's just for the unfurled URLs, then sure